wix/alacritty.wxs

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" 
+    xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+    <Product Name="Alacritty" Id="*" UpgradeCode="87c21c74-dbd5-4584-89d5-46d9cd0c40a7" Language="1033" Codepage="1252" Version="0.2.9" Manufacturer="Alacritty">
```

Comment:
I'm not familiar with WiX, but just to make sure I have read up on it correctly:

The `UpgradeCode` is used as an ID to replace existing installs when a new version is installed, correct? So we will be able to keep this `UpgradeCode` the same until the end of time? Or does this need to be changed every time a new version is released?

Reply:
The `UpgradeCode` should remain the same until the end of time, the `Id` should change with each update but it is set to `Id="*"` so that will be handled automatically.